debezium-core/src/main/java/io/debezium/connector/common/TaskPartition.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.common;
+
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * Describes the partition of the source to be processed by the task and provides its representation
```

Comment:
Isn't one task processing multiple partitions? If so, I think the name is confusing.

Reply:
It is confusing. The class name predates https://github.com/debezium/debezium-design-documents/pull/1. It is a source partition in terms of connector (not to be confused with the Kafka Connect source **partition** (`Map<String,String>`) or topic **partition** (int)).

Should we name it just `Partition`? The FQN should be sufficient to address any confusion. The "source" prefix might be redundant since Debezium is a collection of source connectors.

Is `io.debezium.connector.common` the right package for this interface?